eve/main.yml

```diff
@@ -485,6 +485,7 @@ stages:
           name: Trigger build of previous version
           stage_names:
             - buildprev
+            - build-example-solution
```

Comment:
We could make the stage trigger name generic since we are not building the previous version of the example-solution
e.g `Trigger build for previous version and example-solution stages simultaneously` or something along those lines...

Reply:
yep, done